### PR TITLE
Added entity property theming

### LINKED ISSUE #49: theme_entity_plus_property is not defined
The function `theme_entity_plus_property` doesn't seem to be defined anywhere, despite the direction to `@see theme_entity_plus_property()` in line 13 of `entity_plus.theme.inc`. 

This is function should be the equivalent of Drupal 7's `theme_entity_property`, which is defined in the file `entity.theme.inc`. Is there a reason why it's not included in Backdrop's version of Entity API?

### PR DESCRIPTION
**Pull request by argiepiano**
Follow-up to #49  
This pull request adds theming for entity properties, and is an adaptation of the same functionality in Drupal's Entity API.

### CODE REVIEW

**Comment by laryn** [comment]
@hosef Are you able to review this one?

File: entity_plus.module [hunk 1]
```diff
@@ -227,7 +227,7 @@ function entity_plus_type_supports($entity_plus_type, $op) {
     return in_array('EntityPlusControllerInterface', class_implements($info['controller class']));
   }
   if ($op == 'form') {
-    return (bool) entity_plus_ui_controller($entity_plus_type);
+    return (bool) entity_ui_controller($entity_plus_type);
```
Comment (laryn) [inline entity_plus.module:230]: @hosef @argiepiano This function (`entity_ui_controller`) is provided by the Entity UI module, which is not a hard requirement for Entity Plus. Do we need to check if the module is enabled, or split this out from Entity UI?

Comment (argiepiano) [reply entity_plus.module:230]: @laryn yes, I believe the best approach is to check if Entity UI is enabled. I'll modify

Comment (argiepiano) [reply entity_plus.module:230]: I have modified the code to check for module entity_ui

Comment (laryn) [reply entity_plus.module:230]: Thanks, hopefully either I or @hosef can complete the review of this PR soon and make a new release with your improvements in. Thanks for your work!

**Pull request merged**